--- a/usr/local/petget/0setup	2011-10-22 12:29:17.000000000 +0800
+++ b/usr/local/petget/0setup	2011-10-25 02:58:30.000000000 +0800
@@ -130,9 +130,9 @@
   echo "This will be downloaded from:"
   echo "$PKGLISTURI"
   echo "and will be processed and named $PKGLISTFILE."
-  echo -n "Press ENTER key to download, any other to abort: "
+  echo -n "Press ENTER key to download, any other to skip it: "
   read downloadit
-  [ "$downloadit" != "" ] && exit
+  if [ "$downloadit" = "" ] ; then
   DLFILE="`basename $PKGLISTURI`"
   [ -f $DLFILE ] && mv -f $DLFILE /tmp/${DLFILE}-backup1 #v431 otherwise wget creates a new file ${DLFILE}.1
   rxvt -name pet -bg orange -geometry 80x10 -e wget $PKGLISTURI
@@ -251,6 +251,9 @@
 ...exited from 0setup script." >> /var/woof/0setup_fail_report_$RUNNINGPUP
    exit
   fi
+  else 
+   echo "skipped"
+  fi
  fi
 done
 
@@ -265,14 +268,17 @@
   echo "This will be downloaded from:"
   echo "$PKGLISTURI"
   echo "and will be named $PKGLISTFILE."
-  echo -n "Press ENTER key to download, any other to abort: "
+  echo -n "Press ENTER key to download, any other to skip it: "
   read downloadit
-  [ "$downloadit" != "" ] && exit
+  if [ "$downloadit" = "" ] ; then
   DLFILE="`basename $PKGLISTURI`"
   mv -f $DLFILE /tmp/${DLFILE}-backup1 #v431 otherwise wget creates a new file ${DLFILE}.1
   rxvt -name pet -bg orange -geometry 80x10 -e wget $PKGLISTURI
   sync
   [ -f $DLFILE ] && echo "...success"
+  else 
+   echo "skipped"
+  fi
  fi
 done
 
