--- a/etc/xdg/templates/_root_.jwmrc	2012-02-22 19:40:59.000000000 +0200
+++ b/etc/xdg/templates/_root_.jwmrc	2012-02-24 19:18:58.000000000 +0200
@@ -58,6 +58,7 @@
 	<!-- Additional tray attributes: autohide, width, border, layer, layout -->
 	<!-- Tray definition moved to separate file: .jwmrc-tray -->
 	<Include>/root/.jwmrc-tray</Include>
+ 	<Include>/root/.jwmrc-tray2</Include>
 
 	<!-- Visual Styles -->
 
@@ -120,6 +121,7 @@
 		<IconPath>/usr/local/lib/X11/pixmaps</IconPath>
 		<IconPath>/usr/share/pixmaps</IconPath>
 		<IconPath>/usr/local/share/pixmaps</IconPath>
+		<IconPath>/var/local/icons/</IconPath>
 	<!-- </Icons> -->
 
 	<StartupCommand>
@@ -127,7 +129,20 @@
 
 	<!-- Virtual Desktops -->
 	<!-- Name tags can be contained within Desktops for desktop names.  BK: syntax change with jwm 500... -->
-	<Desktops  width="3" height="1"/>
+	<Desktops  width="2" height="1">
+	<Desktop Name="One">
+	<Background type="tile">/usr/share/backgrounds/default.jpg</Background>
+	</Desktop>
+
+	<Desktop Name="Two">
+	<Background type="tile">/usr/share/backgrounds/default.jpg</Background>
+	</Desktop>
+
+	<Desktop Name="Three">
+	<Background type="tile">/usr/share/backgrounds/default.jpg</Background>
+	</Desktop>
+
+	</Desktops>
 
 	<!-- Double click speed (in milliseconds) -->
 	<DoubleClickSpeed>400</DoubleClickSpeed>
