--- a/powerd	2013-09-11 22:59:05.000000000 +0300
+++ b/powerd	2013-09-11 19:12:04.000000000 +0300
@@ -458,7 +458,7 @@
 XO=;
 CONFIGDIR=/etc/powerd
 DATADIR=$CONFIGDIR
-LIBEXECDIR=/usr/libexec/powerd
+LIBEXECDIR=/usr/bin
 
 VERSIONFILE=$DATADIR/version
 CONFIGFILE=$CONFIGDIR/powerd.conf
@@ -735,7 +735,7 @@
         debug=1
         : @ debug begun, $powerd_version
         cat /proc/version >&2
-        cat /boot/olpc_build >&2
+        #cat /boot/olpc_build >&2
         ;;
     off)
         if [ "$tracing" ]
@@ -965,7 +965,7 @@
 
 log powerd $powerd_version startup at $(date), on XO-$XO
 log $(< /proc/version)
-log olpc build: $(< /boot/olpc_build)
+#log olpc build: $(< /boot/olpc_build)
 
 # make a file pattern match of "foo*" to return nothing if foo* doesn't exist
 shopt -s nullglob
@@ -1001,8 +1001,8 @@
 # pull the (first) wlan interface name from the output of iwconfig.
 # if we find none, wlan may be rfkilled, in which case choose eth0.
 WLANIFACE=$(iwconfig 2>/dev/null |
-    sed -n -e 's/^\(eth\|wlan\)\([0-9]\+\).*/\1\2/p' | sed 1q)
-: ${WLANIFACE:=eth0}
+    sed -n -e 's/^\(eth\|wlan\|mlan\)\([0-9]\+\).*/\1\2/p' | sed 1q)
+: ${WLANIFACE:=mlan0}
 
 
 splash()
@@ -1054,7 +1054,7 @@
     log shutting down due to $*
     leavesplash # kill the splasher (leaving splash visible)
     sleep .05s
-    /sbin/poweroff &
+    /usr/bin/wmpoweroff &
     sleep 9999999
 }
 
@@ -1869,8 +1869,6 @@
     while : snooze loop
     do
 
-        pwrlog_take_reading suspend
-
         trace until-sleep_type is $until-$sleep_type 
 
         case $until-$sleep_type in
@@ -2022,8 +2020,6 @@
         trace got wakeup: $wakeupsource @ $lastwakeup, \
                 slept $(( lastwakeup - sleep_started ))
 
-        pwrlog_take_reading resume-$wakeupsource
-
         # if we don't know what woke us, and it was very close
         # to the alarm time, treat it as such.  (useful esp. during
         # development, when rtc driver doesn't yet support wakeup
@@ -2545,12 +2541,6 @@
     config_CPU_IDLE_LIMIT=10
     config_MONITOR_NETWORK_ACTIVITY=yes
 
-    config_PWRLOG_INTERVAL=0
-    config_PWRLOG_DIR=/home/olpc/power-logs
-    config_PWRLOG_OWNER=olpc:olpc
-    config_PWRLOG_LOGSIZE=50            # Kbytes
-    config_PWRLOG_LOGDIRSIZE=1000       # Kbytes
-
     config_AMBIENT_BRIGHT=50    # low OLS sensor trigger level, for sunshine
     config_AMBIENT_NORMAL=80    # high trigger level, for shade or indoors
     config_NEVER_SLEEP=false
@@ -2926,7 +2916,6 @@
 {
     [ -n "$powerd_dbus_pid" ] && kill $powerd_dbus_pid
     [ -n "$inotify_pid" ] && kill $inotify_pid
-    pwrlog_take_reading shutdown
     set_wakeupevents none
     unsplash
     dcon thaw
@@ -3000,9 +2989,7 @@
             continue
         fi
 
-        pwrlog_take_reading $event-event $arg2 $arg3 $more
-
-        case $event in
+	case $event in
 
         powerbutton|fake_powerbutton)
             # sync, in case the user keeps holding the button.
@@ -3225,30 +3212,6 @@
     done <&6  # from the fifo
 }
 
-configure_pwrlog()
-{
-    if [ $config_PWRLOG_INTERVAL -gt 30 -a \
-            -s $LIBEXECDIR/power-logger \
-            -a -z "$no_battery" ]
-    then
-        pwrlog_inside_powerd=yes
-
-        mkdir -p $config_PWRLOG_DIR
-        chown $config_PWRLOG_OWNER $config_PWRLOG_DIR
-
-        . $LIBEXECDIR/power-logger
-        pwrlog_init $config_PWRLOG_INTERVAL $config_PWRLOG_DIR \
-            $config_PWRLOG_OWNER \
-            $config_PWRLOG_LOGSIZE $config_PWRLOG_LOGDIRSIZE
-
-    else
-        # install a null handler
-        pwrlog_take_reading()
-        {
-            trace pwrlog unconfigured, or unavailable
-        }
-    fi
-}
 
 # powerd runs "set -u".  variables that aren't initialized
 # naturally are set here.
@@ -3292,9 +3255,9 @@
 
 init_netactivity_tracking
 
-configure_pwrlog
+#configure_pwrlog
 
-pwrlog_take_reading startup
+#pwrlog_take_reading startup
 
 sched_unfreezetimer
 
