--- a/powerd	2012-04-25 19:11:08.000000000 +0300
+++ b/powerd	2012-04-25 22:49:16.000000000 +0300
@@ -439,7 +439,7 @@
 XO=;
 CONFIGDIR=/etc/powerd
 DATADIR=$CONFIGDIR
-LIBEXECDIR=/usr/libexec/powerd
+LIBEXECDIR=/usr/bin
 
 VERSIONFILE=$DATADIR/version
 CONFIGFILE=$CONFIGDIR/powerd.conf
@@ -710,8 +710,6 @@
         set -x
         debug=1
         : @ debug begun, $powerd_version
-        cat /proc/version >&2
-        cat /boot/olpc_build >&2
         ;;
     off)
         if [ "$tracing" ]
@@ -769,13 +767,6 @@
 
 read_hwinfo()
 {
-    if [ -x /usr/bin/olpc-hwinfo ]
-    then
-        hwvendor=OLPC
-        hwname=XO
-        hwversion=$(olpc-hwinfo model)
-        return
-    fi
     read hwvendor < /sys/class/dmi/id/sys_vendor ||
         hwvendor="n/a"
     read hwname < /sys/class/dmi/id/product_name ||
@@ -901,8 +901,8 @@
 exec 1>&2   # capture stdout, too. (prevents console spew)
 
 log powerd $powerd_version startup at $(date), on XO-$XO
-log $(< /proc/version)
-log olpc build: $(< /boot/olpc_build)
+#log $(< /proc/version)
+#log olpc build: $(< /boot/olpc_build)
 
 # make a file pattern match of "foo*" to return nothing if foo* doesn't exist
 shopt -s nullglob
@@ -941,7 +929,7 @@
 # if we find none, wlan may be rfkilled, in which case choose eth0
 WLANIFACE=$(iwconfig 2>/dev/null |
     sed -n -e 's/^\(eth\|wlan\)\([0-9]\+\).*/\1\2/p' | sed 1q)
-: ${WLANIFACE:=eth0}
+: ${WLANIFACE:=wlan0}
 
 
 splash()
@@ -993,7 +981,7 @@
     log shutting down due to $*
     leavesplash # kill the splasher (leaving splash visible)
     sleep .05s
-    /sbin/poweroff &
+    /usr/bin/wmpoweroff &
     sleep 9999999
 }
 
@@ -1772,7 +1760,7 @@
     while : snooze loop
     do
 
-        pwrlog_take_reading suspend
+		#pwrlog_take_reading suspend
 
         trace until-sleep_type is $until-$sleep_type 
 
@@ -1935,7 +1923,7 @@
         trace got wakeup: $wakeupsource @ $lastwakeup, \
                 slept $(( lastwakeup - sleep_started ))
 
-        pwrlog_take_reading resume-$wakeupsource
+        #pwrlog_take_reading resume-$wakeupsource
 
         # if we don't know what woke us, and it was very close
         # to the alarm time, treat it as such.  (useful esp. during
@@ -2453,11 +2441,6 @@
     config_CPU_IDLE_LIMIT=10
     config_MONITOR_NETWORK_ACTIVITY=yes
 
-    config_PWRLOG_INTERVAL=0
-    config_PWRLOG_DIR=/home/olpc/power-logs
-    config_PWRLOG_LOGSIZE=50            # Kbytes
-    config_PWRLOG_LOGDIRSIZE=1000       # Kbytes
-
     config_AMBIENT_BRIGHT=50    # low OLS sensor trigger level, for sunshine
     config_AMBIENT_NORMAL=80    # high trigger level, for shade or indoors
 }
@@ -2832,7 +2815,7 @@
 {
     [ -n "$powerd_dbus_pid" ] && kill $powerd_dbus_pid
     [ -n "$inotify_pid" ] && kill $inotify_pid
-    pwrlog_take_reading shutdown
+    #pwrlog_take_reading shutdown
     set_wakeupevents none
     unsplash
     dcon thaw
@@ -2903,8 +2886,8 @@
             continue
         fi
 
-        pwrlog_take_reading $event-event $arg2 $arg3 $more
-
+		#pwrlog_take_reading $event-event $arg2 $arg3 $more
+		
         case $event in
 
         powerbutton|fake_powerbutton)
@@ -3134,30 +3117,6 @@
     done <&6  # from the fifo
 }
 
-configure_pwrlog()
-{
-    if [ $config_PWRLOG_INTERVAL -gt 30 -a \
-            -s $LIBEXECDIR/power-logger \
-            -a -z "$no_battery" ]
-    then
-        pwrlog_inside_powerd=yes
-
-        mkdir -p $config_PWRLOG_DIR
-        chown olpc:olpc $config_PWRLOG_DIR
-
-        . $LIBEXECDIR/power-logger
-        pwrlog_init $config_PWRLOG_INTERVAL $config_PWRLOG_DIR \
-            $config_PWRLOG_LOGSIZE $config_PWRLOG_LOGDIRSIZE
-
-    else
-        # install a null handler
-        pwrlog_take_reading()
-        {
-            trace pwrlog unconfigured, or unavailable
-        }
-    fi
-}
-
 # powerd runs "set -u".  variables that aren't initialized
 # naturally are set here.
 am_ebook=;
@@ -3201,9 +3160,9 @@
 
 init_netactivity_tracking
 
-configure_pwrlog
+#configure_pwrlog
 
-pwrlog_take_reading startup
+#pwrlog_take_reading startup
 
 sched_unfreezetimer
 
