--- a/usr/sbin/snapmergepuppy	2011-05-05 08:43:29.000000000 +0800
+++ b/usr/sbin/snapmergepuppy	2011-10-06 08:21:28.000000000 +0800
@@ -31,6 +31,11 @@
 
 export LANG=C #110206 Dougal: I **think** this should not cause problems with filenames
 
+#exit if we are sleeping
+if [ -f /tmp/sleeping.now ] ; then
+	exit 0
+fi
+
 #variables created at bootup by 'init' script in initramfs...
 . /etc/rc.d/PUPSTATE
 . /etc/DISTRO_SPECS #v412
