--- a/drivers/usb/storage/usb.c	2012-03-25 15:01:56.000000000 +0300
+++ b/drivers/usb/storage/usb.c	2012-03-25 16:03:41.000000000 +0300
@@ -844,6 +844,7 @@
 	}
 	scsi_scan_host(us_to_host(us));
 	dev_dbg(dev, "scan complete\n");
+	printk(KERN_DEBUG "usb-storage: device scan complete\n");
 
 	/* Should we unbind if no devices were detected? */
 
@@ -928,6 +929,7 @@
 {
 	int result;
 	struct device *dev = &us->pusb_intf->dev;
+	printk(KERN_DEBUG "usb-storage: device found at %d\n", us->pusb_dev->devnum);
 
 	/* Make sure the transport and protocol have both been set */
 	if (!us->transport || !us->proto_handler) {
@@ -972,6 +974,8 @@
 
 	if (delay_use > 0)
 		dev_dbg(dev, "waiting for device to settle before scanning\n");
+		printk(KERN_DEBUG "usb-storage: waiting for device "
+				"to settle before scanning\n");
 	queue_delayed_work(system_freezable_wq, &us->scan_dwork,
 			delay_use * HZ);
 	return 0;
